### ISSUE TITLE
New submissions for Wed, 26 Jul 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#67]
# New submissions for Wed, 26 Jul 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### A Comparison of the Composition of Planets in Single- and Multi-Planet  Systems Orbiting M dwarfs
 - **Authors:** Romy Rodríguez Martínez, David V. Martin, B. Scott Gaudi, Joseph G. Schulze, Anusha Pai Asnodkar, Kiersten M. Boley, Sarah Ballard
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2307.13034
 - **Pdf link:** https://arxiv.org/pdf/2307.13034
 - **Abstract**
 We investigate and compare the composition of M-dwarf planets in systems with only one known planet (``singles") to those residing in multi-planet systems (``multis") and the fundamental properties of their host stars. We restrict our analysis to planets with directly measured masses and radii, which comprise a total of 70 planets: 30 singles and 40 multis in 19 systems. We compare the bulk densities for the full sample, which includes planets ranging in size from $0.52 R_{\oplus}$ to $12.8R_\oplus$, and find that single planets have significantly lower densities on average than multis, which we cannot attribute to selection biases. We compare the bulk densities normalized by an Earth model for planets with $R_{p} < 6R_{\oplus}$, and find that multis are also denser with 99\% confidence. We calculate and compare the core/water mass fractions (CMF/WMF) of low-mass planets ($M_p <10 M_{\oplus}$), and find that the likely rocky multis (with $R_p <1.6 R_{\oplus}$) have lower CMFs than singles. We also compare the [Fe/H] metallicity and rotation period of all single versus multi-planet host stars with such measurements in the literature and find that multi-planet hosts are significantly more metal-poor than those hosting a single planet. Moreover, we find that host star metallicity decreases with increasing planet multiplicity. In contrast, we find only a modest difference in the rotation period. The significant differences in planetary composition and metallicity of the host stars point to different physical processes governing the formation of single- and multi-planet systems in M dwarfs.
#### The IceCube Collaboration -- Contributions to the 38th International  Cosmic Ray Conference (ICRC2023)
 - **Authors:** IceCube: R. Abbasi, M. Ackermann, J. Adams, S. K. Agarwalla, J. A. Aguilar, M. Ahlers, J.M. Alameddine, N. M. Amin, K. Andeen, G. Anton, C. Argüelles, Y. Ashida, S. Athanasiadou, S. N. Axani, X. Bai, A. Balagopal V., M. Baricevic, S. W. Barwick, V. Basu, R. Bay, J. J. Beatty, J. Becker Tjus, J. Beise, C. Bellenghi, C. Benning, S. BenZvi, D. Berley, E. Bernardini, D. Z. Besson, E. Blaufuss, S. Blot, F. Bontempo, J. Y. Book, C. Boscolo Meneguolo, S. Böser, O. Botner, J. Böttcher, E. Bourbeau, J. Braun, B. Brinson, J. Brostean-Kaiser, R. T. Burley, R. S. Busse, D. Butterfield, M. A. Campana, K. Carloni, E. G. Carnie-Bronca, S. Chattopadhyay, N. Chau, C. Chen, Z. Chen, D. Chirkin, S. Choi, B. A. Clark, L. Classen, A. Coleman, G. H. Collin, A. Connolly, J. M. Conrad, P. Coppin,  et al. (345 additional authors not shown)
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2307.13047
 - **Pdf link:** https://arxiv.org/pdf/2307.13047
 - **Abstract**
 The IceCube Observatory at the South Pole has been operating in its full configuration since May 2011 with a duty cycle of about 99%. Its main component consists of a cubic-kilometer array of optical sensors deployed deep in the Glacial ice designed for the detection of high-energy astrophysical neutrinos. A surface array for cosmic ray air shower detection, IceTop, and a denser inner subdetector, DeepCore, significantly enhance the capabilities of the observatory, making it a multipurpose facility. This list of contributions to the 38th International Cosmic Ray Conference in Nagoya, Japan (July 26 - August 3, 2023) summarizes the latest results from IceCube covering a broad set of key questions in physics and astrophysics. The papers in this index are grouped topically to highlight IceCube contributions related to high-energy neutrino and multi-messenger astrophysics, cosmic-ray physics, low-energy neutrino transients such as Galactic supernovae, fundamental physics, detector calibration and event reconstruction, education and public outreach, and research and development for the IceCube Upgrade, a scheduled dense sensor infill complemented by calibration devices. Contributions related to IceCube-Gen2, the future extension of IceCube, are available in a separate collection.
#### Non-Equilibrium Abundances Treated Holistically (NEATH): the molecular  composition of star-forming clouds
 - **Authors:** F. D. Priestley, P. C. Clark, S. C. O. Glover, S. E. Ragan, O. Fehér, L. R. Prole, R. S. Klessen
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2307.13072
 - **Pdf link:** https://arxiv.org/pdf/2307.13072
 - **Abstract**
 Much of what we know about molecular clouds, and by extension star formation, comes from molecular line observations. Interpreting these correctly requires knowledge of the underlying molecular abundances. Simulations of molecular clouds typically only model species that are important for the gas thermodynamics, which tend to be poor tracers of the denser material where stars form. We construct a framework for post-processing these simulations with a full time-dependent chemical network, allowing us to model the behaviour of observationally-important species not present in the reduced network used for the thermodynamics. We use this to investigate the chemical evolution of molecular gas under realistic physical conditions. We find that molecules can be divided into those which reach peak abundances at moderate densities ($10^3 \, {\rm cm^{-3}}$) and decline sharply thereafter (such as CO and HCN), and those which peak at higher densities and then remain roughly constant (e.g. NH$_3$, N$_2$H$^+$). Evolving the chemistry with physical properties held constant at their final values results in a significant overestimation of gas-phase abundances for all molecules, and does not capture the drastic variations in abundance caused by different evolutionary histories. The dynamical evolution of molecular gas cannot be neglected when modelling its chemistry.
#### Spirals and clumps in V960 Mon: signs of planet formation via  gravitational instability around an FU Ori star?
 - **Authors:** P. Weber, S. Pérez, A. Zurlo, J. Miley, A. Hales, L. Cieza, D. Principe, M. Cárcamo, A. Garufi, Á. Kóspál, M. Takami, J. Kastner, Z. Zhu, J. Williams
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2307.13433
 - **Pdf link:** https://arxiv.org/pdf/2307.13433
 - **Abstract**
 The formation of giant planets has traditionally been divided into two pathways: core accretion and gravitational instability. However, in recent years, gravitational instability has become less favored, primarily due to the scarcity of observations of fragmented protoplanetary disks around young stars and low occurrence rate of massive planets on very wide orbits. In this study, we present a SPHERE/IRDIS polarized light observation of the young outbursting object V960 Mon. The image reveals a vast structure of intricately shaped scattered light with several spiral arms. This finding motivated a re-analysis of archival ALMA 1.3 mm data acquired just two years after the onset of the outburst of V960 Mon. In these data, we discover several clumps of continuum emission aligned along a spiral arm that coincides with the scattered light structure. We interpret the localized emission as fragments formed from a spiral arm under gravitational collapse. Estimating the mass of solids within these clumps to be of several Earth masses, we suggest this observation to be the first evidence of gravitational instability occurring on planetary scales. This study discusses the significance of this finding for planet formation and its potential connection with the outbursting state of V960 Mon.
#### Retrieving the 21-cm signal from the Epoch of Reionization with learnt  Gaussian process kernels
 - **Authors:** Florent G. Mertens, Jérôme Bobin, Isabella P. Carucci
 - **Subjects:** Cosmology and Nongalactic Astrophysics (astro-ph.CO); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2307.13545
 - **Pdf link:** https://arxiv.org/pdf/2307.13545
 - **Abstract**
 Direct detection of the Cosmic Dawn and Epoch of Reionization via the redshifted 21-cm line of neutral Hydrogen will have unprecedented implications for studying structure formation in the early Universe. This exciting goal is challenged by the difficulty of extracting the faint 21-cm signal buried beneath bright astrophysical foregrounds and contaminated by numerous systematics. Here, we focus on improving the Gaussian Process Regression (GPR) signal separation method originally developed for LOFAR observations. We address a key limitation of the current approach by incorporating covariance prior models learnt from 21-cm signal simulations using Variational Autoencoder (VAE) and Interpolatory Autoencoder (IAE). Extensive tests are conducted to evaluate GPR, VAE-GPR, and IAE-GPR in different scenarios. Our findings reveal that the new method outperforms standard GPR in component separation tasks. Moreover, the improved method demonstrates robustness when applied to signals not represented in the training set. It also presents a certain degree of resilience to data systematics, highlighting its ability to effectively mitigate their impact on the signal recovery process. However, our findings also underscore the importance of accurately characterizing and understanding these systematics to achieve successful detection. Our generative approaches provide good results even with limited training data, offering a valuable advantage when a large training set is not feasible. Comparing the two algorithms, IAE-GPR shows slightly higher fidelity in recovering power spectra compared to VAE-GPR. These advancements highlight the strength of generative approaches and optimise the analysis techniques for future 21-cm signal detection at high redshifts.

by olozhika (Xing Yuchen). 

2023-07-26
